tests/integration/loss/test_perceptual.py

```diff
@@ -5,8 +5,11 @@
 
 from pystiche import enc, loss, ops
 
+from tests.utils import suppress_deprecation_warning
+
 
 class TestPerceptualLoss:
+    @suppress_deprecation_warning
```

Comment:
Why did you put the decorator here? IIRC, this should not be deprecated. Same for `set_style_image`.

Reply:
I think `FeatureReconstructionOperator` is deprecated.

```
FAILED tests/integration/loss/test_perceptual.py::TestPerceptualLoss::test_set_content_image - UserWarning: The class pystiche.ops.FeatureReco...
FAILED tests/integration/loss/test_perceptual.py::TestPerceptualLoss::test_set_style_image - UserWarning: The class pystiche.ops.FeatureRecons...
```

Will have to disable warnings as failures to get the full log for warnings. (will update post dinner)

Reply:
> I think `FeatureReconstructionOperator` is deprecated.

Yes, but in such a case we should fix the test. We can simply replace `FeatureReconstructionOperator` with `FeatureReconstructionLoss` and should eliminate the warning.

Reply:
Alright, sure! I'll fix this.

Reply:
This has been fixed now!